docs/app-engine.md

```diff
@@ -77,8 +77,14 @@ gcloud app deploy --project=wptdashboard index.yaml queue.yaml dispatch.yaml
 `index.yaml`, `queue.yaml` or `dispatch.yaml`.)
 
 To deploy manually, follow the same instructions as production but replace
-`wptdashboard` with `wptdashboard-staging`, and use `make deploy_staging`
-instead of `make deploy_production`.
+`wptdashboard` with `wptdashboard-staging`, use `make deploy_staging`
+instead of `make deploy_production` and use `app.staging.yaml` instead of `app.yaml`:
+
+```sh
+wptd_exec_it make deploy_production PROJECT=wptdashboard APP_PATH=webapp/web/app.staging.yaml
```

Comment:
The makefile target should be `deploy_staging` and PROJECT should be `wptdashboard-staging`

This will match with what the CI [job](https://github.com/web-platform-tests/wpt.fyi/blob/2916e0e6eed2cc1b36628699a094218985cddd9c/.github/workflows/deploy.yml#L42) does 
https://github.com/web-platform-tests/wpt.fyi/blob/2916e0e6eed2cc1b36628699a094218985cddd9c/util/deploy-staging.sh#L47-L49

Reply:
oops late-night PR. Thanks for catching it!